.github/workflows/on-pullrequest-dockerfile-ac.yml

```diff
@@ -0,0 +1,95 @@
+  # SPDX-License-Identifier: Apache-2.0
+  # Licensed to the Ed-Fi Alliance under one or more agreements.
+  # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+  # See the LICENSE and NOTICES files in the project root for more information.
+
+  name: On Pull Request - Dockerfile - Admin Console
+
+  on:
+    push:
+      branches:
+        - adminapi23-rc.*
```

Comment:
Should the branch change?

Should the name of the workflow change? Since it is not triggering "On Pull Request".